tasks/task1.scala

```diff
@@ -1 +1,30 @@
 // TODO: task 1
+object Main extends App {
+  val generateValues = (1 to 50).toList
+  def sumForLoop(ints: List[Int]) = {
+    var sum: Int = 0
+    for (int <- ints) {
+      sum = sum + int
+    }
+    sum
+  }
+  def sumRecursion(values: List[Int], partialSum: Int): Int = {
```

Comment:
Have `partialSum` with a default, so that we can call `sumRecursion` with only the list as an argument?
`partialSum: Int = 0`